ksql-engine/src/main/java/io/confluent/ksql/codegen/SqlToJavaVisitor.java

```diff
@@ -145,7 +145,9 @@ private String formatExpression(final Expression expression) {
         final StringLiteral node,
         final Void context
     ) {
-      return new Pair<>("\"" + node.getValue() + "\"", Schema.OPTIONAL_STRING_SCHEMA);
+      return new Pair<>(
+          "\"" + node.getValue().replace("\"", "\\\"") + "\"",
```

Comment:
You might want to look at `StringEscapeUtils` which handles all the weird escape sequences you might need.

Reply:
Hi Guys, `{"ksql": "CREATE STREAM stream1 WITH (PARTITIONS=4, REPLICAS=1, KAFKA_TOPIC='stream1_topic', VALUE_FORMAT='AVRO') AS SELECT * FROM TestMessagePingPong_AVRO_STREAM WHERE \"operator_id\" = 1154;","streamsProperties":{"ksql.streams.auto.offset.reset":"latest"}}  ` this gives me   `"message": "io.confluent.ksql.util.KsqlStatementException: Field 'TESTMESSAGEPINGPONG_AVRO_STREAM.OPERATOR_ID' cannot be resolved. `

version Ksql server version 5.4.0

Is this related to this feature?

Reply:
@EddieJamsession - please create a new GitHub issue and we can help you out there! Makes it easier for us to track.